pom.xml

```diff
@@ -270,6 +270,29 @@
                 <javadoc.doclint.none>-Xdoclint:none</javadoc.doclint.none>
             </properties>
         </profile>
+        <profile>
+            <id>dev</id>
+            <build>
```

Comment:
Got it. That definitely looks more neat.

I used the same technique for `maven-invoker-plugin` and `maven-archetype-plugin`. Unfortunately I didn't manage to find a `skip` property for `maven-shade-plugin`.